apps/src/netsim/NetSimTabsComponent.html.ejs

```diff
@@ -8,6 +8,10 @@
   var showMyDevice = shouldShowTab(level, NetSimTabType.MY_DEVICE);
   var showRouter = shouldShowTab(level, NetSimTabType.ROUTER);
   var showDns = shouldShowTab(level, NetSimTabType.DNS);
+
+  var instructionsContent = netsimMsg[level.instructionsLocKey] ?
```

Comment:
There's already a localization pipeline for level types that inherit from the `Blockly` model.  You'll probably want to re-use this to get instructions piped through as `appOptions.level.instructions`: https://github.com/code-dot-org/code-dot-org/blob/e65f7a381db4a0e6e54bfc85429f72979c1e8f3d/dashboard/app/helpers/levels_helper.rb#L253